### DISCUSSION TITLE
Is it Possible to Configure Vert.X gateway for MySQL ?

### DISCUSSION THREAD

**Discussion by shekarnode** [apiman/apiman#1040] [Q&A]
Hi ,

I have been following  the steps mentioned in [git book](https://apiman.gitbooks.io/apiman-installation-guide/content/installation-guide/vertx/download.html) to setup Vert.X gateway.

I am able to setup and run vert.X gateway with ES config file using following command :
`java -jar apiman-gateway.jar  -conf configs/conf-es.json`

Now , I have a requirement to use Vert.X gateway in production Environment with  MySQL DB .
So is it possible to establish a connection to MySQL DB from Vert.X gateway.
If so , can you share the post or any article  that would help me to achieve it .

**Comment by msavy** [comment]
Hi @shekarnode. What have you tried so far?

**Reply by volkflo** [reply]
Is this related to this issue: https://github.com/apiman/apiman/issues/975?
